core/examples/luigi-sample-angular/src/assets/sampleconfig.js

```diff
@@ -341,16 +341,28 @@ Luigi.setConfig({
         {
           pathSegment: 'ext',
           label: 'External Page',
+          loadingIndicator: {
+            enabled: true,
```

Comment:
I would remove `enabled: true` here and two times more below, since this is what we consider already as default value.